spock-core/core.gradle

```diff
@@ -35,11 +37,7 @@ dependencies {
 
 jar {
   manifest {
-    name = 'spock-core'
     attributes(
-      'Created-By': "${System.properties['java.version']} (${System.properties['java.vendor']} ${System.properties['java.vm.version']})".toString(),
-      'Build-Date': buildDate,
-      'Build-Time': buildTime,
```

Comment:
@leonard84 It was removed due to compatibility issues or you just don't think it is worth anymore?

Reply:
Removed so that the jar are stable, i.e. don't change just because you build it, when the actual content is still the same.

Reply:
In that case you can also remove settings those 2 variables in the main `build.gradle`.